tests/integration/components/user-organizations-list-test.js

```diff
@@ -49,12 +53,12 @@ test('with several organizations renders all required elements', function(assert
     username: 'JoshSmith'
   });
 
-  this.render(hbs`{{user-organizations-list user=user organizations=organizations}}`);
+  page.render(hbs`{{user-organizations-list user=user organizations=organizations}}`);
 
-  assert.equal(this.$('.user-organizations-list').length, 1, 'Component\'s element is rendered');
-  assert.equal(this.$('h2').text(), 'Organizations', 'The header renders');
-  assert.equal(this.$('li').length, 3, 'All organization items render');
-  assert.equal(this.$('li:first h3').text().trim(), 'Organization 1', 'The organization header renders');
-  assert.equal(this.$('li:first p').text().trim(), 'Organization 1 description', 'The organization description renders');
+  assert.ok(page.userOrganisationListVisible);
+  assert.equal(page.header, 'Organizations', 'The header renders');
+  assert.equal(page.listItemCount, 3, 'All organization items render');
+  assert.equal(page.header, 'Organization 1', 'The organization header renders');
+  assert.equal(page.description, 'Organization 1 description', 'The organization description renders');
   assert.equal(this.$('li:first img').attr('src'), '/icon_1.png', 'The organization image renders');
```

Comment:
I see http://ember-cli-page-object.js.org/docs/v1.6.x/api/properties#property, just wanted to confirm if that's the same as `attr`?